src/Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -19,11 +20,24 @@ namespace Stratis.Bitcoin.Configuration.Logging
     /// </summary>
     public class ExtendedLoggerFactory : LoggerFactory
     {
+        private const string NLogConfigFileName = "NLog.config";
+
         /// <summary>Configuration of console logger.</summary>
         public ConsoleLoggerSettings ConsoleSettings { get; set; }
 
         /// <summary>Provider of console logger.</summary>
         public ConsoleLoggerProvider ConsoleLoggerProvider { get; set; }
+
+        /// <summary>Loads an NLog.config file from the <see cref="DataFolder"/>, if it exists</summary>
+        public void LoadNLogConfiguration(DataFolder dataFolder)
+        {
+            if (dataFolder == null)
+                return;
+
+            string configPath = Path.Combine(dataFolder.RootPath, NLogConfigFileName);
```

Comment:
Maybe inside a try catch ? unless we are OK with a bad config file or IO issue to mess things up ...

Reply:
We either would need to crash the node or to display some message to the user. The worst thing would be to silently get the exception and discard it.